xcube edit metadata tool (#202)
dcs4cop/xcube: CHANGES.md

### Enhancements

- Added new CLI tool `xcube edit` and API function `xcube.api.edit_metadata`

>   which allows to edit the metadata of an existing xcube dataset. (#170)

Reviewer: ```suggestion
  which allows editing the metadata of an existing xcube dataset. (#170)
```

> - `xcube serve` now recognises xcube datasets with consolidated

Reviewer: ```suggestion
- `xcube serve` now recognises xcube datasets with
```

>   metadata. (#141)

Reviewer: ```suggestion
  metadata consolidated by the `xcube opmimize` command. (#141)
```
